CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## unreleased
```

Comment:
you don't need to update this file for this PR. I'll make an update in the release PR

Reply:
No problem, I've reverted the change in this file